legend-engine-xt-persistence-cloud-pure/src/main/resources/core_persistence_cloud/external/format/awspersistence/metamodel.pure

```diff
@@ -0,0 +1,104 @@
+// Copyright 2022 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import meta::pure::persistence::metamodel::*;
+import meta::legend::service::metamodel::*;
+import meta::relational::metamodel::relation::*;
+import meta::pure::alloy::connections::alloy::authentication::*;
+import meta::pure::alloy::connections::alloy::specification::*;
+import meta::pure::persistence::metamodel::trigger::*;
+import meta::pure::persistence::metamodel::notifier::*;
+import meta::pure::persistence::metamodel::persister::*;
+import meta::pure::persistence::metamodel::persister::deduplication::*;
+import meta::pure::persistence::metamodel::persister::ingestmode::*;
+import meta::pure::persistence::metamodel::persister::audit::*;
+import meta::pure::persistence::metamodel::persister::transactionmilestoning::*;
+import meta::pure::runtime::*;
+
+import meta::relational::metamodel::*;
+
+Class meta::external::persistence::aws::cnas::spec::ModelClassProperty
```

Comment:
Not sure about the "cnas" package. Do we expect other "cnas" stuff to be added to legend-engine ?